yb-voyager/src/migassessment/sizing.go

```diff
@@ -639,32 +641,33 @@ func getExperimentFile(assessmentMetadataDir string) (string, error) {
 		}
 	}
 	if !fetchedFromRemote {
-		err := os.WriteFile(assessmentMetadataDir+experimentDataFileName, experimentData20240, 0644)
+		err := os.WriteFile(filepath.Join(AssessmentMetadataDir, EXPERIMENT_DATA_FILENAME), experimentData20240, 0644)
 		if err != nil {
 			return "", fmt.Errorf("failed to write experiment data file: %w", err)
 		}
 	}
-	return assessmentMetadataDir + experimentDataFileName, nil
+
+	return filepath.Join(AssessmentMetadataDir, EXPERIMENT_DATA_FILENAME), nil
 }
 
-func checkAndDownloadFileExistsOnRemoteRepo(assessmentMetadataDir string) (bool, error) {
+func checkAndDownloadFileExistsOnRemoteRepo() (bool, error) {
 	// check if the file exists on remote github repository using the raw link
-	remotePath := GITHUB_RAW_LINK + experimentDataFileName
+	remotePath := GITHUB_RAW_LINK + EXPERIMENT_DATA_FILENAME
 	resp, err := http.Get(remotePath)
 	if err != nil {
 		return false, fmt.Errorf("failed to make GET request: %w", err)
 	}
 	defer func() {
 		if closingErr := resp.Body.Close(); closingErr != nil {
-			err = closingErr
+			log.Infof("failed to close the response body for GET api")
```

Comment:
My bad if i suggested `Infof` log level for this.
I think at least `Warn` level is what we should do at least, or even `Error` level should be fine, so that it is noticeable while going the logs.

Reply:
Done. Changed to Warn log level. I don't think Error level is necessary as it does not impact the actual flow of the execution.